### ISSUE TITLE
tslib cannot be found

### ISSUE THREAD

**Issue by thepalebluedot** [awayjs/core#11]
Typescript compilation yields the following error:

lib/errors/AbstractMethodError.ts(7,34): error TS2354: This syntax requires an imported helper but module 'tslib' cannot be found.

**Comment by rob-bateman** [comment]
You need to uncomment the tslib alias in the webpack config your using

**Comment by thepalebluedot** [comment]
Can you add a link to the line in the code?

**Comment by rob-bateman** [comment]
https://github.com/awayjs/awayjs-examples/blob/master/webpack.config.js#L76

**Comment by thepalebluedot** [comment]
Right, I'm compiling typescript in this module directly (core), not on the examples.
That is, I'm executing "tsc" or "npm run build" in @awayjs/core, not in @awayjs/awayjs-examples.
